src/AbiBinProvider-node.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+
+const fs = require('fs');
+const path = require('path');
+
+const loadContracts = (provider, abiFolderPath, binFolderPath) => {
+  if (!path.isAbsolute(abiFolderPath)) {
+    const err = new Error(
+      '"abiFolderPath" is not Absolute. Please provide absolute path.',
+    );
+    throw err;
+  }
+  if (!path.isAbsolute(binFolderPath)) {
+    const err = new Error(
+      '"binFolderPath" is not Absolute. Please provide absolute path.',
+    );
+    throw err;
+  }
+
+  // add all ABIs from abiFolderPath
+  fs.readdirSync(abiFolderPath).forEach((abiFile) => {
+    const contractName = path.basename(abiFile, path.extname(abiFile));
+    const contractAbi = JSON.parse(fs.readFileSync(abiFile));
+    provider.addABI(contractName, contractAbi);
+  });
```

Comment:
don't we use the `mosaic-contracts` npm package?

Reply:
Yes, but the consumers in brandedtoken.js and openst.js still assume that the contracts can be loaded from file system via that interface. I'm not sure if we should add a deprecation notice for that already?